.vscode/extentions.json

```diff
@@ -0,0 +1,11 @@
+{
+	"recommendations": [
+		"quarto.quarto",
+		"streetsidesoftware.code-spell-checker",
+		"vivaxy.vscode-conventional-commits",
+		"pshaddel.conventional-branch",
+        "jebbs.plantuml",
```

Comment:
```suggestion
```

We won't use Puml here, not necessary.

Reply:
Well, I have already made some pumls for e.g., explaining commits, branches, the GitHub flow, etc.?

Reply:
I'd like to avoid including those and rather use Mermaid, since the other courses also use them and since they integrate nicely with natively using the themes of the website.